llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -550,7 +550,14 @@ static bool isSupportedInstr(const MachineInstr &MI) {
   // 11.12. Vector Widening Integer Multiply Instructions
   // FIXME: Add support for 11.12 instructions
   // 11.13. Vector Single-Width Integer Multiply-Add Instructions
```

Comment:
One concern that is not related to the code: the [RVV repo](https://github.com/riscv/riscv-v-spec) has been archived, and all the content are put in a section of [riscv-isa-manual](https://github.com/riscv/riscv-isa-manual) now, so these section markers are kind of misleading.

Reply:
Good observation, thanks. I have updated the patch.